self-tests-staging.yml

```diff
@@ -6,8 +6,8 @@ containers:
     working_dir: /
     image: "hadjri/evg-e2e-test-ubuntu"
     resources:
-      cpu: 4096
-      memory_mb: 8192
+      cpu: 2048
+      memory_mb: 3584
```

Comment:
Per our discussion on Slack, I don't think we need to reduce the CPU/memory available to the container since the staging cluster has large enough container instances.